stats/test/platform-routes.test.js

```diff
@@ -527,14 +561,27 @@ const givenMonthlyActiveStationCount = async (pgPoolEvaluate, month, stationCoun
 }
 
 const givenDailyRewardTransferMetrics = async (pgPoolStats, day, transferStats) => {
-  await pgPoolStats.query(`
-    INSERT INTO daily_reward_transfers (day, to_address, amount, last_checked_block)
-    SELECT $1 AS day, UNNEST($2::text[]) AS to_address, UNNEST($3::int[]) AS amount, UNNEST($4::int[]) AS last_checked_block
-    ON CONFLICT DO NOTHING
+  for (const transfer of transferStats) {
+    const participantResult = await pgPoolStats.query(
+      'SELECT id FROM participants WHERE participant_address = $1',
+      [transfer.toAddress]
+    )
+
+    if (participantResult.rows.length === 0) {
+      throw new Error(`Participant address ${transfer.toAddress} not found`)
+    }
```

Comment:
Let's use `mapParticipantsToIds` to map new addresses to participant ids, so that we don't have to do it manually in tests.